Gemfile

```diff
@@ -77,6 +77,8 @@ gem 'devise'
 
 gem 'cancancan'
 
```

Comment:
- [ ] Your effort in creating the migrations to add references to the post, comment, and likes tables is commendable. However, it seems there is an issue with the current order of your migration files. The migration process in Ruby starts from the first migration and proceeds in chronological order. The first migration you have is attempting to add a reference to the post table, but the post table has not been created yet.

  To resolve this issue, it's essential to ensure that all table creations are completed before any alterations that involve adding references to those tables. This way, the necessary tables will exist in the database before attempting to establish foreign key relationships.

  **To fix the problem, follow these steps:**
  In the image below, all the migrations that I label "A" Should come before all those I label with "B".

  **_Create Migrations for Table Creation:_** Ensure that you have migration files to create the post, comment, and likes tables before any migrations that involve adding references to them.

<img width="899" alt="image" src="https://github.com/Mikey1-nuel/Blog-app/assets/84156714/207a337e-7e4e-452d-8fda-6b9c8b8295c7">